Clients/src/presentation/tools/riskCalculator.ts

```
@@ -0,0 +1,98 @@
+      return { level: "Very Low Risk", color: RISK_LABELS.noRisk.color };
+    } else if (score <= 8) {
+      return { level: "Low Risk", color: RISK_LABELS.low.color };
+    } else if (score <= 12) {
+      return { level: "Medium Risk", color: RISK_LABELS.medium.color };
+    } else if (score <= 16) {
+      return { level: "High Risk", color: RISK_LABELS.high.color };
+    } else {
+      return { level: "Very High Risk", color: RISK_LABELS.critical.color };
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)